.github/workflows/update-attributions.yml

```diff
@@ -39,7 +39,7 @@ jobs:
             -f content='+1'
         env:
           COMMENT_ID: ${{ github.event.comment.id }}
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.LAVAMOAT_UPDATE_TOKEN }}
```

Comment:
Nit: We should create an issue or add a TODO comment for renaming this token

Reply:
I just created this issue https://github.com/MetaMask/metamask-extension/issues/25737